apps/portal/js/designer.js

```diff
@@ -2509,7 +2533,7 @@ $(function () {
                     $('#banner-data').val(dataUrl);
                 }
             });
-            
+
             $('.ues-banner-placeholder button').prop('disabled', false);
```

Comment:
using removeProp will remove the disabled attr instead of setting the value to false. This is not recommended in special attr like disabled, checked and selected. http://api.jquery.com/removeProp/